util/drakeUtil.h

```diff
@@ -45,6 +46,45 @@ template <typename Derived> inline void destroyDrakeMexPointer(const mxArray* mx
 //  mexPrintf(mexIsLocked() ? "mex file is locked\n" : "mex file is unlocked\n");
 }
 
+template <typename DerivedA>
+mxArray* eigenToMatlab(const DerivedA &m)
```

Comment:
Well I suppose you know my opinion on this by now. I can do the explicit instantiation thing again, but it just eats up my time with nothing to show for it in my view.